tools/mpremote/mpremote/main.py

```diff
@@ -228,6 +229,26 @@ def argparse_mip():
     return cmd_parser
 
 
+def argparse_romfs():
+    cmd_parser = argparse.ArgumentParser(description="manage ROM partitions")
+    _bool_flag(cmd_parser, "mpy", "m", True, "download as compiled .mpy files (default)")
+    cmd_parser.add_argument(
+        "--partition",
+        "-p",
+        type=int,
+        default=0,
+        help="ROM partition to use",
+    )
+    cmd_parser.add_argument(
+        "--output",
+        "-o",
+        help="output file",
+    )
+    cmd_parser.add_argument("command", nargs=1, help="romfs command (e.g. query, build, deploy)")
```

Comment:
```suggestion
    cmd_parser.add_argument("command", nargs=1, help="romfs command. Supported commands: query, build, deploy.")
```

(As this is the only place the tool actually prints the list of accepted commands, I think.)

The other way to improve this would be to add another argparse sub-command parser so we get generated list of commands and their associated options, but that seems like something that can be done later when there are more sub-commands.

Reply:
I struggled a bit with this sub-command parsing.  Was trying to copy how `fs <subcommand>` works.

Reply:
Help message adjusted.